.template/variants/web/app/javascript/build.js

```diff
@@ -6,6 +6,7 @@ require('esbuild')
     inject: ['app/javascript/global.js'],
     bundle: true,
     sourcemap: true,
+    minify: !watch,
```

Comment:
do we have any unit test for this and the `.template/variants/web/package.json.rb` bellow @sanG-github

Reply:
Thank @andyduong1920, I added test for `package.json` in 963968 ✅ But not sure how to write tests for this file 🤔

Reply:
yeah that it ok @sanG-github